From 6c9496d048c4c3534524ea3212c780776844f025 Mon Sep 17 00:00:00 2001
From: timsong-cpp <rs2740@gmail.com>
Date: Fri, 25 Jul 2025 09:03:35 -0500
Subject: [PATCH] Handle \bot

---
 RawDocument.hs | 2 +-
 Render.hs      | 1 +
 2 files changed, 2 insertions(+), 1 deletion(-)

diff --git a/RawDocument.hs b/RawDocument.hs
index c1f82bc..935878c 100644
--- a/RawDocument.hs
+++ b/RawDocument.hs
@@ -195,7 +195,7 @@ storeCmds =
 			"phi int theta zeta FlushAndPrintGrammar break backslash centering " ++
 			"normalbaselineskip land lor mapsto normalfont textmu tablerefname figurerefname newline " ++
 			"obeyspaces bnfindent vdots tabcolsep columnbreak emergencystretch commentellip " ++
-			"gamma widowpenalties sffamily parskip left right `")
+			"gamma widowpenalties sffamily parskip left right bot `")
 	, (1, "hspace footnote textit textrm textnormal texttt textbf ensuremath ref ref* mbox bibitem mathop " ++
 			"terminal literalterminal noncxxterminal textsl textsc textsf text term overline " ++
 			"tcode noncxxtcode literaltcode footnotetext microtypesetup cline mathtt mathit mathrm mathsf " ++
diff --git a/Render.hs b/Render.hs
index 4b34cb6..2df6e74 100644
--- a/Render.hs
+++ b/Render.hs
@@ -128,6 +128,7 @@ simpleMacros =
 	, (">"              , "&#9;")
 	, ("bnfindent"      , "&emsp;&emsp;&emsp;")
 	, ("\n"             , "\n")
+	, ("bot"            , "⊥")
 	]
 	++ [(n, Text.pack [c]) | (n, c) <- greekAlphabet]
 
-- 
2.43.0

