From 411eddd3f44b0a28cc959a5fec3e7087e9c7fafe Mon Sep 17 00:00:00 2001
From: timsong-cpp <rs2740@gmail.com>
Date: Tue, 15 Aug 2023 23:30:52 -0500
Subject: [PATCH] update for assets

---
 RawDocument.hs | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/RawDocument.hs b/RawDocument.hs
index 859a6ee..db4c7b5 100644
--- a/RawDocument.hs
+++ b/RawDocument.hs
@@ -273,7 +273,7 @@ parseFile macros =
 
 loadFigure :: Text -> Text
 loadFigure f = unsafePerformIO $ do
-		dot <- readFile p
+		dot <- readFile ("assets/" ++ p)
 		svg <- readProcess "dot" ["-Tsvg",
 			"-Gbgcolor=transparent",
 			"-Gsize=8",
-- 
2.34.1

