From f6d3a19e638a466655018f506228a550efb5d7e6 Mon Sep 17 00:00:00 2001
From: timsong-cpp <rs2740@gmail.com>
Date: Wed, 11 Jan 2017 09:20:43 -0500
Subject: [PATCH] Fix paths for my box

---
 Render.hs       | 2 +-
 SectionPages.hs | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/Render.hs b/Render.hs
index e2d0cee..8dbe38b 100644
--- a/Render.hs
+++ b/Render.hs
@@ -557,7 +557,7 @@ memodRenderMath = memo2 $ \s inline -> unsafePerformIO $ do
 	let args = ["--inline" | inline] ++ ["--", cleanup s]
 	formula <- Text.replace " focusable=\"false\"" "" 
 		. rmTrailingNewline -- Prevents artifacts in [rand.adapt.ibits]#4
-		. Text.pack . readProcess "/usr/lib/node_modules/mathjax-node/bin/tex2html" args ""
+		. Text.pack . readProcess "/usr/local/lib/node_modules/mathjax-node/bin/tex2html" args ""
 	return $ if inline then formula else "</p><p style='text-align:center'>" ++ formula ++ "</p><p>"
 	where
 		cleanup = Text.unpack . Text.replace "\\discretionary{}{}" "" . Text.pack
diff --git a/SectionPages.hs b/SectionPages.hs
index 3638623..5ea049b 100644
--- a/SectionPages.hs
+++ b/SectionPages.hs
@@ -192,5 +192,5 @@ writeCssFile = do
 				".MJXc-TeX-math-I {font-style: italic}"
 		-- Replace fonts to make sure code in formulas matches code in code blocks, etc.
 	mjx <- replaceFonts . Text.pack .
-		readProcess "/usr/lib/node_modules/mathjax-node/bin/tex2html" ["--css", ""] ""
+		readProcess "/usr/local/lib/node_modules/mathjax-node/bin/tex2html" ["--css", ""] ""
 	writeFile (outputDir ++ "/14882.css") (base ++ mjx)
-- 
2.7.4

