From 68c342b4ecfd396110bda583b133d1755252a43a Mon Sep 17 00:00:00 2001
From: timsong-cpp <rs2740@gmail.com>
Date: Wed, 20 Oct 2021 21:52:16 -0500
Subject: [PATCH] handle textasciitilde

---
 Document.hs    | 1 +
 RawDocument.hs | 2 +-
 2 files changed, 2 insertions(+), 1 deletion(-)

diff --git a/Document.hs b/Document.hs
index b468fe1..046b3cd 100644
--- a/Document.hs
+++ b/Document.hs
@@ -264,6 +264,7 @@ indexKeyContent = mconcat . map ikc
 		ikc (TeXComm "}" _ []) = "}"
 		ikc (TeXComm "caret" _ []) = "^"
 		ikc (TeXComm "tilde" _ []) = "~"
+		ikc (TeXComm "textasciitilde" _ []) = "~"
 		ikc (TeXComm "^" _ []) = "^"
 		ikc (TeXComm "\"" _ []) = "\""
 		ikc (TeXComm "" _ []) = ""
diff --git a/RawDocument.hs b/RawDocument.hs
index 30422f7..882d64d 100644
--- a/RawDocument.hs
+++ b/RawDocument.hs
@@ -191,7 +191,7 @@ storeCmds =
 			"phi int theta zeta FlushAndPrintGrammar break backslash centering " ++
 			"normalbaselineskip land lor mapsto normalfont textmu tablerefname figurerefname newline " ++
 			"obeyspaces bnfindent vdots tabcolsep columnbreak emergencystretch commentellip " ++
-			"gamma widowpenalties sffamily parskip left right")
+			"gamma widowpenalties sffamily parskip left right textasciitilde")
 	, (1, "hspace footnote textit textrm textnormal texttt textbf ensuremath ref mbox " ++
 			"terminal literalterminal noncxxterminal renontermdef textsl textsc textsf text term " ++
 			"tcode noncxxtcode literaltcode footnotetext microtypesetup cline mathtt mathit mathrm mathsf " ++
-- 
2.25.1

