From ef408d47d2b0e6f25e3b556cec741655c0018ed8 Mon Sep 17 00:00:00 2001
From: timsong-cpp <rs2740@gmail.com>
Date: Thu, 5 Jan 2017 09:14:23 -0500
Subject: [PATCH] Fix path for my box

---
 Render.hs       | 2 +-
 SectionPages.hs | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/Render.hs b/Render.hs
index 66000ab..eea87b2 100644
--- a/Render.hs
+++ b/Render.hs
@@ -535,7 +535,7 @@ memodRenderMath :: String -> Bool -> Text
 memodRenderMath = memo2 $ \s inline -> unsafePerformIO $ do
 	let args = ["--inline" | inline] ++ [s]
 	formula <- Text.replace " focusable=\"false\"" "" . Text.pack .
-		readProcess "/usr/lib/node_modules/mathjax-node/bin/tex2html" args ""
+		readProcess "/usr/local/lib/node_modules/mathjax-node/bin/tex2html" args ""
 	return $ if inline then formula else "</p><p style='text-align:center'>" ++ formula ++ "</p><p>"
 
 renderTable :: LaTeX -> [Row [Element]] -> RenderContext -> Text
diff --git a/SectionPages.hs b/SectionPages.hs
index 3638623..5ea049b 100644
--- a/SectionPages.hs
+++ b/SectionPages.hs
@@ -192,5 +192,5 @@ writeCssFile = do
 				".MJXc-TeX-math-I {font-style: italic}"
 		-- Replace fonts to make sure code in formulas matches code in code blocks, etc.
 	mjx <- replaceFonts . Text.pack .
-		readProcess "/usr/lib/node_modules/mathjax-node/bin/tex2html" ["--css", ""] ""
+		readProcess "/usr/local/lib/node_modules/mathjax-node/bin/tex2html" ["--css", ""] ""
 	writeFile (outputDir ++ "/14882.css") (base ++ mjx)
-- 
2.7.4

