From 4f9bc8a594c4d24fdaba261a97567964911c1f6e Mon Sep 17 00:00:00 2001
From: timsong-cpp <rs2740@gmail.com>
Date: Sun, 23 Mar 2025 18:03:20 -0500
Subject: [PATCH] Highlight __has_embed

---
 Render.hs | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Render.hs b/Render.hs
index c61d766..a2b4cde 100644
--- a/Render.hs
+++ b/Render.hs
@@ -337,7 +337,7 @@ highlight ctx x
     -- keywords
     | (a, x') <- texSpan p x, a /= "" = (case () of
         _ | a `elem` keywords -> spanTag "keyword"
-        _ | a `elem` ["defined", "__has_include", "__has_cpp_attribute", "_Pragma"] -> spanTag "preprocessordirective"
+        _ | a `elem` ["defined", "__has_include", "__has_embed", "__has_cpp_attribute", "_Pragma"] -> spanTag "preprocessordirective"
         _ | a `elem` ["nullptr", "true", "false"] -> spanTag "literal"
         _ | otherwise -> id) (render (TeXRaw a) ctx) ++ highlight ctx x'
     where p c = isAlphaNum c || c == '_'
-- 
2.43.0

